lnrpc/routerrpc/config_active.go

```diff
@@ -19,6 +19,19 @@ type Config struct {
 	// directory, named DefaultRouterMacFilename.
 	RouterMacPath string `long:"routermacaroonpath" description:"Path to the router macaroon"`
 
+	// MinProbability is the minimum required route probability to attempt
+	// the payment.
+	MinProbability float64 `long:"minprobability" description:"Minimum required route probability to attempt the payment"`
+
+	// PenaltyHalfLife defines after how much time a penalized node or
+	// channel is back at 50% probability.
+	PenaltyHalfLife int `long:"penaltyhalflife" description:"Defines the duration in minutes after which a penalized node or channel is back at 50% probability"`
```

Comment:
should be okay to use `time.Duration` directly?

Reply:
didn't know that. works ok indeed. changed.